.github/workflows/ci-idris2.yml

```diff
@@ -321,13 +320,16 @@ jobs:
           path: ~/.idris2/
       - name: Install build dependencies
         run: |
-          brew install chezscheme
-          brew install coreutils
+          brew install chezscheme gmp
           echo "$HOME/.idris2/bin" >> $GITHUB_PATH
           chmod +x $HOME/.idris2/bin/idris2 $HOME/.idris2/bin/idris2_app/*
       - name: Build self-hosted
         run: make all IDRIS2_BOOT="idris2 -Werror" && make install
         shell: bash
+      - name: Install test dependencies
+        run: |
+          brew install minimal-racket
+          raco pkg install --auto compiler-lib r6rs-lib
```

Comment:
Why are we installing racket for the chezscheme test?

Reply:
It was more to prove that the Racket backend tests work on the CI (there are no Racket tests enabled otherwise — currently the windows build is the only use of Racket).

Seemed like a good idea at the time. With `ubuntu-bootstrap-racket` disabled, I'd say it still is.

Reply:
It'd be tidier to have a separate job. The test prefix makes things a bit awkward, or can it just run `make -C tests only=racket`?

Reply:
I have removed this workflow change.